packages/elina/elina.1.1/opam

```diff
@@ -34,7 +36,9 @@ depends: [
   "conf-perl"
   "apron"
 ]
+flags: light-uninstall
```

Comment:
This flag as well as the “remove” section are no longer necessary with opam 2

Reply:
Thanks for the feedback, let m try to fix it and send another request.